### ISSUE TITLE
"npm test" failed

### ISSUE THREAD

**Issue by gerisse** [rocket-pool/rocketpool#183]
hi
After `npm install`, i run `npm test `. It failed , with this message :
`Contract: RocketNodeManager
       admin: can set a node's trusted status:
     Error: value out-of-bounds (argument="_index", value="-1", code=INVALID_ARGUMENT, version=abi/5.0.0-beta.153)`
an idea ? thanks

**Comment by darcius** [comment]
Hey @gerisse! Are you getting this error on master branch? I just ran the tests on master and they all passed for me.

What version of node are you running? You can find that out by typing `node -v` in terminal.

**Comment by gerisse** [comment]
yes, Darcius, i 'm on master branch. `node -v` gives `v15.0.1` . Thanks

**Comment by darcius** [comment]
@gerisse Ah ok, found the issue! We hadn't tested on > Node 13 LTS before, so I upgraded to Node 14 LTS which caused issues with our versions of Ganache / Truffle. Updated those and then encountered your error which it seems was due to a bug in that unit test which was being ignored in previous node versions.

If you do a fresh pull + `npm install && npm run test`, it should all work for you now :+1:

**Comment by gerisse** [comment]
ok , so i did the update , and choosed the node 14 version. But the same issue appears :-(

**Comment by darcius** [comment]
That is odd, it appears fixed for me. Did you run  a `git pull` before running the tests again?

**Comment by gerisse** [comment]
yes, i pulled it before
